src/test/java/task30/ByVariables.java

```diff
@@ -0,0 +1,13 @@
+package task30;
+
+import org.openqa.selenium.By;
+
+public class ByVariables {
+    public static By SignIn = By.className("authorization-link");
+    public static By EmailInput = new By.ById("email");
+    public static By PasswordInput = new By.ById("pass");
+    public static By SignInButton = new By.ById("send2");
+
+    public static By AssertElement = By.className("logged-in");
```

Comment:
follow java coding conventions. names of the variables, methods should be conform lower-case format